raiden/raiden_event_handler.py

```diff
@@ -59,6 +59,7 @@
 from raiden.transfer.views import get_channelstate_by_token_network_and_partner
 from raiden.utils.packing import pack_balance_proof_update, pack_withdraw
 from raiden.utils.typing import MYPY_ANNOTATION, Address, BlockSpecification, Nonce
+from raiden_contracts.tests.utils.constants import LOCKSROOT_OF_NO_LOCKS
```

Comment:
This import is wrong as the constant is added in the wrong placce.

```
from raiden_contracts.tests.utils.constants import LOCKSROOT_OF_NO_LOCKS
```
You're importing the contract's test module into the codebase.

Reply:
1. I'll define the constant in `raiden.utils.constants` and use it in this PR.
2. I'll move `LOCKSROOT_OF_NO_LOCKS` from `raiden-contracts.tests.utils.constants` into `raiden-contracts.constants`
3. I'll release raiden-contracts 0.24.0
4. I'll change `raiden` so that it imports from `raiden-contracts.constants`.

Reply:
The import is now from `raiden.utils.constants`.